### PR TITLE
UI and storybook updates

### PR DESCRIPTION
**Pull request by georgewrmarshall**
## Explanation
Some UI updates

### Before
Screen Shot 2022-06-28 at 2 29 48 PM

### After
Screen Shot 2022-06-28 at 2 22 08 PM

Also updating and adding some stories
Screen Shot 2022-06-28 at 2 19 31 PM
Screen Shot 2022-06-28 at 2 20 09 PM
Screen Shot 2022-06-28 at 2 21 07 PM

### CODE REVIEW

File: ui/components/app/collectible-default-image/collectible-default-image.js [hunk 1]
```diff
@@ -13,19 +13,12 @@ export default function CollectibleDefaultImage({
   const t = useI18nContext();
   return (
     <div
-      className={classnames('collectible-default__wrapper', {
-        'collectible-default__clickable': handleImageClick,
+      className={classnames('collectible-default', {
+        'collectible-default--clickable': handleImageClick,
       })}
```
Comment (georgewrmarshall) [inline ui/components/app/collectible-default-image/collectible-default-image.js:18]: Updating classes here to follow BEM naming convention

File: ui/components/app/collectible-default-image/collectible-default-image.js [hunk 2]
```diff
@@ -13,19 +13,12 @@ export default function CollectibleDefaultImage({
...
     >
-      <img className="collectible-default__image" src="images/empty-nft.png" />
-      <Typography
-        variant={TYPOGRAPHY.H7}
-        className={classnames('collectible-default__text', {
-          'collectible-default__clickable': handleImageClick,
-        })}
-        onClick={handleImageClick}
-      >
+      <Typography variant={TYPOGRAPHY.H7} className="collectible-default__text">
```
Comment (georgewrmarshall) [inline ui/components/app/collectible-default-image/collectible-default-image.js:21]: Removing click handler from text

File: ui/components/app/collectible-default-image/collectible-default-image.stories.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+export const handleImageClick = (args) => (
+  <div style={{ width: 200, height: 200 }}>
+    <CollectibleDefaultImage {...args} />
+  </div>
+);
+
+handleImageClick.args = {
+  // eslint-disable-next-line no-alert
+  handleImageClick: () => window.alert('CollectibleDefaultImage clicked!'),
+};
```
Comment (georgewrmarshall) [inline ui/components/app/collectible-default-image/collectible-default-image.stories.js:42]: Adding story for empty image card

File: ui/components/app/collectible-details/collectible-details.stories.js [hunk 4]
```diff
@@ -9,23 +18,24 @@ export default {
...
+export const NoImage = (args) => {
+  return <CollectibleDetails {...args} />;
+};
+
+NoImage.args = {
+  collectible: {
+    ...collectible,
+    image: undefined,
+  },
 };
```
Comment (georgewrmarshall) [inline ui/components/app/collectible-details/collectible-details.stories.js:41]: Adding another story to display the collectible details story when there is no image

File: ui/components/app/collectibles-items/collectibles-items.js [hunk 5]
```diff
@@ -202,7 +202,11 @@ export default function CollectiblesItems({
                   key={`collectible-${i}`}
                   className="collectibles-items__item-wrapper"
                 >
-                  <Card padding={0} justifyContent={JUSTIFY_CONTENT.CENTER}>
+                  <Card
+                    padding={0}
+                    justifyContent={JUSTIFY_CONTENT.CENTER}
+                    className="collectibles-items__item-wrapper__card"
+                  >
```
Comment (georgewrmarshall) [inline ui/components/app/collectibles-items/collectibles-items.js:209]: Adding overflow hidden to the card so the image being contained doesn't overlap the border.

**Review by adonesky1 — APPROVED** [review]
LGTM, the only thing I think we ought to do is line break the collection name -> tokenId so that you can atleast see the beginning of the tokenId in the card:
Screen Shot 2022-06-28 at 4 59 16 PM

**Comment by georgewrmarshall** [comment]
@adonesky1 updated https://github.com/MetaMask/metamask-extension/pull/15071/commits/bc25abd3eb6fc1860adbea33a943b9fb4c255d8f

Screen Shot 2022-06-29 at 8 26 11 AM

**Pull request merged**